x-pack/plugins/alerting/server/task_runner/rule_action_helper.ts

```diff
@@ -82,7 +82,9 @@ export const getSummaryActionsFromTaskState = ({
   summaryActions?: ThrottledActions;
 }) => {
   return Object.entries(summaryActions).reduce((newObj, [key, val]) => {
-    const actionExists = actions.some((action) => generateActionHash(action) === key);
+    const actionExists = actions.some(
+      (action) => action.frequency?.summary && generateActionHash(action) === key
```

Comment:
As we change the hash for summary actions, this is not needed but added anyway just to double check...